pages/docs/coding-conventions/search-inputs.en-US.mdx

```diff
@@ -0,0 +1,93 @@
+# Search inputs
```

Comment:
Perhaps mention preference for server-side search over client-side search filtering if possible, as the latter means having to download the entire data set (and handling server-side pagination correctly) and is inefficient.

Reply:
Happy to get a line from you showing example usage of the new server-side fetching utility functions @chibongho.

Reply:
Thanks!